sidebar/observer.c

```diff
@@ -84,7 +90,8 @@ static bool calc_divider(struct SidebarWindowData *wdata)
     }
   }
 
-  const bool changed = (width != wdata->divider_width);
+done:
+  changed = (width != wdata->divider_width);
```

Comment:
Can `changed` be initialized here?

Reply:
Oddly, no.
I'm sure `gcc` complained too, but I can't reproduce it today.
However, clang complains:
```
sidebar/observer.c:93:3: error: expected expression
  bool changed = (width != wdata->divider_width);
```

It's not happy to initialise a variable immediately after a label.
Putting `{}`s around the last 4 lines solves this, but looks ugly.